libraries/botbuilder-dialogs/src/memory/dialogPath.ts

```diff
@@ -13,6 +13,9 @@ export class DialogPath {
     /// Currently expected properties.
     public static readonly expectedProperties = 'dialog.expectedProperties';
 
+    /// Default operation to use for entities where there is no identified operation entity.
+    public static readonly defaultOperation = 'dialog.defaultOperation';
```

Comment:
>defaultOperation [](start = 54, length = 16)

Does this need to be $defaultOperation?  I ask because that is the key in adaptiveDialog.

Reply:
> > defaultOperation [](start = 54, length = 16)
> 
> Does this need to be $defaultOperation? I ask because that is the key in adaptiveDialog.

I don't think so, we have a private property in adaptiveDialog named _defaultOperationKey_ which represents the key $defaultOperation, which is a part of schema.